.ruff.toml

```diff
@@ -0,0 +1,15 @@
+# TODO: Move to pyproject.toml once ready
+target-version = "py39"
+line-length = 80
+extend-exclude = [
+  "cupy/array_api",
+  "docs/source",
+  "tests/cupy_tests/array_api_tests",
+  "third_party",
+]
+
+[lint]
+select = ["E", "F", "W"]
+ignore = [
+  "E501", # line-too-long (let formatters handle this)
```

Comment:
Is it ok to ignore this?

Reply:
It is OK if `ruff format` is present, but seems to be not with `autopep8`.
Let me restore it.

Reply:
Fixed in 3ac295ef9e11a6f4f315456e0ec6d77db201cb8c .